examples/circuitChat.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="//unpkg.com/circuit-sdk@latest/circuit.min.js" async></script>
+    <script type="module" src="../circuit-chat.js" defer></script>
+
+    <style>
+      /* Global page styles which are inherited by element */
+      body {
+        font-family: sans-serif;
+        font-size: 13px;
+      }
+
+      input {
+        width: 150px;
+        margin: 5px 0px 10px 0px;
+      }
+
+      .conversation-button-container { 
+        padding-bottom: 20px;
+        display: flex;
+        align-content: center;
+      }
+
+      .conv-btn {
+        margin-left: 20px;
+      }
+      circuit-chat {
+        /* Width of name in items feed*/
+        --name-width: 80px;
+
+        /* Padding of the item content */
+        --item-padding: 0 0 0 20px;
+      }
+
+      circuit-chat.hidden {
+        display: none;
+      }
+
+      /* Style of the send button */
+      circuit-chat::part(btn) {
+        float: right;
+        padding-left: 10px;
+        margin-top: 15px;
+      }
+
+      /* Style of the input field */
+      circuit-chat::part(inp) {
+        width: 300px;
+        margin-top: 15px;
+      }
+
+      /* Main container */
+      circuit-chat::part(main-container) {
+        width: 700px;
+      }
+
+      /* Conversation Title */
+      circuit-chat::part(conversation-title) {
+        padding-bottom: 10px;
+        font-size: 24px;
+      }
+
+      /* Style of the conversation feed */
+      circuit-chat::part(conversation-container) {
+          height: 550px;
+          border: 1px solid green;
+          overflow-y: auto;
+      }
+
+      /* Style of each conversation item in the feed */
+      circuit-chat::part(conversation-item) {
+        padding:  10px 5px;
+        border-top: 1px solid #dadada;
+        display: flex;
+        align-items: center;
+        word-wrap: break-word;
+      }
+    </style>
+  </head>
+  <body>
+    <h2>Circuit Conversation</h2>
+    <p>Uses web components <code>circuit-chat</code> to retrieve the conversation feed of the <code>target</code> attribute.</p>
+    <div>
+      Enter a Conversation Id to display the feed.<br>
+      <input placeholder="conversation id">
+    </div>
+    <div class="conversation-button-container">
+        <div>Get Conversation Feed</div>
+        <button class="conv-btn" id="get-conv">Click</button>
```

Comment:
We don't use "-" in ids. Use something like "selectConv"

Reply:
done